apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/service/ConsumerService.java

```diff
@@ -322,6 +327,7 @@ private ConsumerToken generateConsumerToken(Consumer consumer, Date expires) {
 
     ConsumerToken consumerToken = new ConsumerToken();
     consumerToken.setConsumerId(consumerId);
+    consumerToken.setLimitCount(portalConfig.openApiLimitCount());
```

Comment:
The rate limit should be set on the UI page, with a default value that users can modify.